### ISSUE TITLE
Coin Transfer

### ISSUE THREAD

**Issue by orobogenius** [StayBusy/StayBusySlim#47]
User can transfer coin / sell coins/ etc.

**Comment by orobogenius** [comment]
Buying and selling coins, should this be a market place feature?

Example story:
User wants coins, user goes to the coins purchase section of the market place and selects **Buy from Users** and can buy from any user.

**Buy from Users** would list all sell ads that has been put up by other users.

**Comment by xyluz** [comment]
Yes. I don't think I follow the question... are you saying it should not be this way?

**Comment by orobogenius** [comment]
I was suggesting one of the ways in which it can be realized.

**Comment by xyluz** [comment]
Yes. So, it means users can put posts out saying they want to sell / buy coins for anyone who is interested... yeah, i think that can work.
